addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -170,4 +170,24 @@
 		<state min="0" max="360" step="1" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="scene">
+		<item-type>Number</item-type>
+		<label>Scene</label>
+		<description>Scene channel</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Scene 1</option>
```

Comment:
Let's go into details then.
This PR implements the following:
1. Added “Livolo On/Off 1-10” subtype in Lighting 5 which corresponds to the function group in RFXmngr.
![image](https://cloud.githubusercontent.com/assets/16033193/21762373/0006878e-d661-11e6-978f-b56d76138655.png)
2. Within the new subtype scene support was implemented:
![image](https://cloud.githubusercontent.com/assets/16033193/21762464/6a6fa2fe-d661-11e6-841c-2b2d23b4f1e5.png)
    a. Scene channel accepts numeric value 0..3 to trigger corresponding scene. Options were added for convenience, option naming is aligned with the RFXCOM Manager application. 
    b. Scene Switch functionality was implemented to address one of the Livolo protocol limitations.
The problem is that for Livolo, ON command:
`    public enum Commands {
        OFF(0x00),
        ON(0x01),`
is in fact a toggle command, meaning that if switch is in ON state, this command will turn it off. While OFF command is "all OFF" command which turns off all switches synchronized to the same (remote control) device ID. That's being said, there is no pure ON/OFF commands for Livolo.
Since scene code is associated with the switch state in the moment of synchronization/learning, we can program e.g. Scene 1 to turn on the switch and Scene 2 to turn it off accordingly. This way our Scene Switch will be acting as a normal switch similar to existing Command channel with the difference of correct behavior regardless of the switch state.